cli/tests/integrations/test_marathon_pod.py

```diff
@@ -85,14 +88,19 @@ def test_pod_update_from_stdin_force_true():
 
 def _pod_add_from_file(file_path):
     cmd = _POD_ADD_CMD + [file_path]
-    return exec_command(cmd)
+    outputs = exec_command(cmd)
+
+    watch_all_deployments()
+    return outputs
```

Comment:
Just for this method, `_pod_add_from_file()`? Or everywhere I'm using `watch_all_deployments()`? I'd prefer not to have to remember to call it each time.